apollo/resolvers.js

```diff
@@ -129,6 +129,23 @@ const resolvers = {
         ...(category_id && { category_id }),
       }
 
+      return submissions.find(conditions)
+    },
+    my_submissions: async (parent, args, context, info) => {
```

Comment:
Could we try to add this specific use cases to the submissions query with the user as another argument? We could then still check, that the user can only filter for his own submissions. I guess it would leave us with a cleaner api

Reply:
Great suggestion! I have made this change.